frontend/ui/data/optionsutil.lua

```diff
@@ -153,10 +154,10 @@ function optionsutil.showValues(configurable, option, prefix, document, is_size)
         end
     else
         local unit
-        if is_size == 1 then
-            unit = G_reader_settings:nilOrTrue("metric_length") and "mm" or "\""
-        else
+        if is_size == "pt" then
```

Comment:
May be the variable should be renamed to "unit" or "size_type" now (as is_size sounds like it should be a boolean true/false).

Reply:
Yes, this will make the code shorter and more readable